internal/command/deploy/machines.go

```diff
@@ -407,10 +407,31 @@ func (md *machineDeployment) setMachineGuest(vmSize string, vmCPUKind string, vm
 	return nil
 }
 
+func (md *machineDeployment) deployMethodIsCompatible(method string) error {
+	switch strings.ToLower(method) {
```

Comment:
Was it case-insensitive?

Reply:
when we deploy, we use ToUpper on the strategy, so we accept values of any case.
https://github.com/superfly/flyctl/blob/d614f1ad9bd5ca729dc2de186cbf0f326d206819/internal/command/deploy/machines.go#L436

SCREAMING_CASE string constants are pretty ugly so I opted to check against lowercase ones, but it shouldn't make a difference :)

Reply:
I see. I wanted to save future Fly customers from having UPPER_CASE vs. lower_case debate, but seems the ship has sailed :)